sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -15,3 +15,6 @@ select replace('abc', 'b');
 
 -- uuid
 select length(uuid()), (uuid() <> uuid());
+
+-- position
+select position('bar', 'foobarbar'), position('bar', 'foobarbar', 5), position(null, 'foobarbar'), position('aaads', null);
```

Comment:
The syntax of position is `POSITION(substr IN str)`. It is different from `LOCATE`. You need to change the parser to support it.